### DISCUSSION TITLE
can i use djl to training paddleocr model？

### DISCUSSION THREAD

**Discussion by xuhongxin** [deepjavalibrary/djl#2874] [Development]
can i use djl training paddleocr model？for example , with paddleocr i use three models:
1. https://resources.djl.ai/test-models/paddleOCR/mobile/det_db.zip
2. https://resources.djl.ai/test-models/paddleOCR/mobile/cls.zip
3. https://resources.djl.ai/test-models/paddleOCR/mobile/rec_crnn.zip
can i training them use my local dataset and generate a new model?

**Comment by zachgk** [comment]
Currently, DJL only supports training with MXNet and PyTorch. So, you can try doing OCR in one of those engines. Or, you can do the paddle training in Python and then use DJL for inference with the fine-tuned model
